### PR TITLE
Undocumented declaration found

### PR DESCRIPTION
**Pull request by LimesKey**
Solves #239 

Signed-off-by: LimesKey <85136735+LimesKey@users.noreply.github.com>

### CODE REVIEW

**Review by Verisimilitude11 — CHANGES_REQUESTED** [review]
This does not solve the issue; it must be implemented via Javadocs.

**Comment by LimesKey** [comment]
Ok let me try again, I only attempted this issue because it's been sitting for a while and thought i'd give it a try.

**Review by Verisimilitude11 — APPROVED** [review]
